static/app/components/onboardingWizard/taskConfig.tsx

```diff
@@ -291,6 +293,21 @@ export function getOnboardingTasks({
       actionType: 'app',
       location: `/organizations/${organization.slug}/projects/new/`,
       display: true,
+      SupplementComponent: withApi(({task}: FirstEventWaiterProps) => {
+        if (hasQuickStartUpdatesFeature(organization)) {
+          if (!projects?.length || task.requisiteTasks.length > 0 || taskIsDone(task)) {
+            return null;
+          }
+          return (
+            <EventWaitingIndicator
+              text={t('Waiting for error')}
+              hasQuickStartUpdatesFeature
+            />
+          );
+        }
+
+        return null;
```

Comment:
nit: I would flip this logic so it's easier to read and needs less indenting
```suggestion
        if (!hasQuickStartUpdatesFeature(organization)) {
          return null;
        }
        if (!projects?.length || task.requisiteTasks.length > 0 || taskIsDone(task)) {
          return null;
        }
        return (
          <EventWaitingIndicator
            text={t('Waiting for error')}
            hasQuickStartUpdatesFeature
          />
        );
```